api/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -362,4 +362,8 @@ may be a introduced as a stand alone extension.
      than linked list (experimental).
   * Revision 0.9.3, 2024-09-05
   ** Rename `CL_MUTABLE_DISPATCH_PROPERTIES_ARRAY_KHR` to
+     `CL_MUTABLE_COMMAND_PROPERTIES_ARRAY_KHR` (provisional).
```

Comment:
I don't think this merge conflict is resolved correctly should be "(experimental)", i.e. as on the last line of this file.